modules/mig/variables.tf

```diff
@@ -163,6 +163,7 @@ variable "subnetwork_project" {
 
 variable "named_ports" {
   description = "Named name and named port. https://cloud.google.com/load-balancing/docs/backend-service#named_ports"
-  type        = "list"
+  type        = list(map(string))
```

Comment:
This should be typed as `list(map(number))`.

Reply:
Done